README.rst

```diff
@@ -123,13 +123,13 @@ https://doi.org/10.7717/peerj-cs.55
 Contact
 =======
 
-To report an issue with PyMC3 or to suggest a feature please use the `issue tracker <https://github.com/pymc-devs/pymc3/issues>`__.
+We are using `discourse.pymc.io <https://discourse.pymc.io/>`__ as our main communication channel. You can also follow us on `Twitter @pymc_devs <https://twitter.com/pymc_devs>`__ for update and other announcements. Our `pymc Gitter channel <https://gitter.im/pymc-devs/pymc>`__ is still open if you want to have a quick interaction with PyMC3 developers and users.
```

Comment:
I dont know about the Gitter - any long discussion we will direct them to discourse anyway. I guess it's still useful if ppl want to ask for quick advice?